presidio-api/cmd/presidio-api/docs/swagger.yaml

```diff
@@ -0,0 +1,1890 @@
+      via GRPC
+    properties:
+      name:
+        description: The streams job name
+        type: string
+        x-go-name: Name
+      streamsRequest:
+        $ref: '#/definitions/StreamRequest'
+    type: object
+    x-go-package: github.com/microsoft/presidio/presidio-api/cmd/presidio-api/docs/vendor/github.com/Microsoft/presidio-genproto/golang
+  StreamsJobResponse:
+    type: object
+    x-go-package: github.com/microsoft/presidio/presidio-api/cmd/presidio-api/docs/vendor/github.com/Microsoft/presidio-genproto/golang
+  StreamsJobTemplate:
+    description: The kuberenetes job template. Creates a job that creates streams
+      containers
+    properties:
+      analyzeTemplateId:
+        description: The analyzer template id configures the fields to analyze
+        type: string
+        x-go-name: AnalyzeTemplateId
+      anonymizeTemplateId:
+        description: The anonymizer template id configures how to anonymize the sensitive
+          data [optional]
+        type: string
+        x-go-name: AnonymizeTemplateId
+      datasinkTemplateId:
+        description: The datasink template id configures the output destination of
+          the analyzer/anonymizer results
+        type: string
+        x-go-name: DatasinkTemplateId
+      description:
+        description: The job description
+        type: string
+        x-go-name: Description
+      name:
+        description: The job name
+        type: string
+        x-go-name: Name
+      streamsTemplateId:
+        description: The scan template id configures job input source
+        type: string
+        x-go-name: StreamsTemplateId
+    type: object
+    x-go-package: github.com/microsoft/presidio/presidio-api/cmd/presidio-api/docs/vendor/github.com/Microsoft/presidio-genproto/golang
+  Transformation:
+    description: Transformation represents the transformation types - how the sensitive
+      text will change
+    properties:
+      fPEValue:
+        $ref: '#/definitions/FPEValue'
+      hashValue:
+        $ref: '#/definitions/HashValue'
+      maskValue:
+        $ref: '#/definitions/MaskValue'
+      redactValue:
+        $ref: '#/definitions/RedactValue'
+      replaceValue:
+        $ref: '#/definitions/ReplaceValue'
+    type: object
+    x-go-package: github.com/microsoft/presidio/presidio-api/cmd/presidio-api/docs/vendor/github.com/Microsoft/presidio-genproto/golang
+  Trigger:
+    description: Trigger defines the trigger for a new job to start
+    properties:
+      schedule:
+        $ref: '#/definitions/Schedule'
+    type: object
+    x-go-package: github.com/microsoft/presidio/presidio-api/cmd/presidio-api/docs/vendor/github.com/Microsoft/presidio-genproto/golang
+info:
+  description: Documentation for Presidio API Swagger.
+  title: Presidio.
+  version: "1.0"
+paths:
+  /analyzer/recognizers/:
+    get:
+      description: Get all recognizers
+      operationId: getAllRecognizersRequest
```

Comment:
The operationId contains "Request" so when you generate code with swagger-codegen, the method name is "GetAllRecognizersRequest" instead of "GetAllRecognizers".

Reply:
Removed "Request" from the end of IDs